lib/cocoapods/installer/xcode/pods_project_generator/pod_target_integrator.rb

```diff
@@ -25,8 +25,8 @@ def initialize(target_installation_result)
           def integrate!
             UI.section(integration_message) do
               target_installation_result.test_specs_by_native_target.each do |test_native_target, test_specs|
-                add_embed_frameworks_script_phase(test_native_target)
-                add_copy_resources_script_phase(test_native_target)
+                add_embed_frameworks_script_phase(test_native_target, test_specs.first)
```

Comment:
I think this might be wrong when I first did it. I dont think we should be passing only the `test_specs.first` here.